.github/actions/constants.ts

```diff
@@ -0,0 +1,16 @@
+type Browser = 'chrome' | 'firefox' | 'safari';
+
+const BROWSERS: Browser[] = ['chrome', 'firefox', 'safari'];
```

Comment:
This way we only need to define it in one place.
```suggestion
const BROWSERS = ['chrome', 'firefox', 'safari'] as const;
export type Browser = typeof BROWSERS[number]
```

Reply:
ok yeah exporting